Tests/ACKLocalizationCoreTests/ACKLocalization+Plurals.swift

```diff
@@ -0,0 +1,88 @@
+import Combine
+import XCTest
+@testable import ACKLocalizationCore
+
+class AuthAPIServiceMock: AuthAPIServicing {
+    func fetchAccessToken(serviceAccount: ServiceAccount) -> AnyPublisher<AccessToken, RequestError> {
+        Empty().eraseToAnyPublisher()
+    }
+}
+
+class SheetsAPIServiceMock: SheetsAPIServicing {
```

Comment:
just a nitpick, if `ACKLocalizationPluralsTests` is `final` I would also make this class final